src/main/webapp/resources/js/components/lists/BasicList.jsx

```diff
@@ -0,0 +1,36 @@
+/**
+ * Component to render a basic list with just a
+ * title and description for each item in the list
+ */
+
+import React from "react";
+import PropTypes from "prop-types";
+import { List, Typography } from "antd";
+
+const { Text } = Typography;
+
+export function BasicList({
+  itemLayout = "horizontal",
+  dataSource = { dataSource },
+  ...props
+}) {
+  return (
+    <List
+      itemLayout="horizontal"
+      dataSource={dataSource}
+      renderItem={item => (
+        <List.Item>
+          <List.Item.Meta
+            title={<Text strong>{item.title}</Text>}
```

Comment:
title's are already bold? Why are you adding `strong`?

Reply:
Unfortunately the titles aren't bolded by default :(